index.html

```diff
@@ -0,0 +1,487 @@
+                    <p>SpaceX</p>
+                    <h3>After SpaceX Starlink Launch, a Fear of Satellites That Outnumber All Visible Stars</h3>
+                    <span class="link-box-date">June 1</span>
+                </div>
+                <div class="link-box">
+                    <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="space">
+                    <p>SpaceX</p>
+                    <h3>After SpaceX Starlink Launch, a Fear of Satellites That Outnumber All Visible Stars</h3>
+                    <span class="link-box-date">June 1</span>
+                </div>
+                <div class="link-box row3">
+                    <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="space">
+                    <p>SpaceX</p>
+                    <h3>After SpaceX Starlink Launch, a Fear of Satellites That Outnumber All Visible Stars</h3>
+                    <span class="link-box-date">June 1</span>
+                </div>
+                <div class="link-box row3">
+                    <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="space">
+                    <p>SpaceX</p>
+                    <h3>After SpaceX Starlink Launch, a Fear of Satellites That Outnumber All Visible Stars</h3>
+                    <span class="link-box-date">June 1</span>
+                </div>
+                <div class="link-box row3">
+                    <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="space">
+                    <p>SpaceX</p>
+                    <h3>After SpaceX Starlink Launch, a Fear of Satellites That Outnumber All Visible Stars</h3>
+                    <span class="link-box-date">June 1</span>
+                </div>
+            </div>
+            <div class="editors-pick">
+                <hr>
+                <span><strong>Editor's Picks</strong></span>
+                <div class="link-box">
+                    <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="space">
+                    <p>SpaceX</p>
+                    <h3>The Covenant is Rising</h3>
+                    <span class="link-box-date">June 1</span>
+                </div>
+                <div class="link-box">
+                    <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="space">
+                    <p>SpaceX</p>
+                    <h3>The Covenant is Rising</h3>
+                    <span class="link-box-date">June 1</span>
+                </div>
+                <div class="link-box">
+                    <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="space">
+                    <p>SpaceX</p>
+                    <h3>The Covenant is Rising</h3>
+                    <span class="link-box-date">June 1</span>
+                </div>
+            </div>
+
+        </div>
+        <div class="subinfo side-adjust" style="padding-bottom: 50px;">
```

Comment:
The footer must start here. Use `<footer>` tag instead of `<div>`. Don't forget to remove the navbar from this container and put inside the header.

Reply:
Hello, i have so.. the footer must contain the links that are way down the page? (the news, living, subscribe, etc.. part)

Reply:
Yes. It must include everything till the end of the page.